lib/wanda/messaging/adapters/amqp/consumer.ex

```diff
@@ -24,13 +21,10 @@ defmodule Wanda.Messaging.Adapters.AMQP.Consumer do
   def consumer_tag, do: "wanda"
 
   @impl GenRMQ.Consumer
-  def handle_message(%GenRMQ.Message{payload: payload} = message) do
-    with {:ok, event} <- Contracts.from_event(payload),
-         :ok <- Policy.handle_event(event) do
-      GenRMQ.Consumer.ack(message)
-    else
-      {:error, reason} ->
-        handle_error(message, reason)
+  def handle_message(%GenRMQ.Message{payload: _payload} = message) do
+    case processor().process(message) do
+      :ok -> GenRMQ.Consumer.ack(message)
```

Comment:
Usually the mix formatter puts the case implementation in a new line. Did you execute `mix format`?

Reply:
Interesting. I did run it before committing and ran it just now and it stayed the same 🤔 I guess this is a corner-case when the `case` is the outermost block in a function?

Reply:
both are valid. If you add a new line just after the arrow it will format the second part of the blocks in the same way

Reply:
Okey!!